drivers/usb/device/usb_dc_stm32.c

```diff
@@ -261,11 +257,6 @@ static int usb_dc_stm32_clock_enable(void)
 
 #endif /* RCC_HSI48_SUPPORT / LL_RCC_USB_CLKSOURCE_NONE */
```

Comment:
The comment should be `/* RCC_CFGR_OTGFSPRE / RCC_CFGR_USBPRE */`

Reply:
Fixed, Txs